src/component/utils/isEventHandled.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule isEventHandled
+ * @typechecks
+ * @flow
+ */
+
+'use strict';
+
+import type {DraftHandleValue} from 'DraftHandleValue';
+
+/**
+ * Utility method for determining whether or not the value returned
+ * from a handler indicates that it was handled.
+ */
+function isEventHandled(value: DraftHandleValue): boolean {
+  return value === 'handled';
+}
+
+module.exports = isEventHandled;
```

Comment:
@hellendag I used commonjs here because somehow it couldn't find the module when using ES6 modules. Is there anything special going on which prevents ES6 modules from working for normal code and not just types?